src/js/components/init_share_dropdown.js

```diff
@@ -0,0 +1,28 @@
+export const initShareDropdown = () => {
```

Comment:
I reckon we should  put guard clauses on these as we've done for the js files above no?

Reply:
Good question, I think that If you have an iteration that's not needed because it will just iterate on an empty array, right? Not sure if it's still good practice to do it tho.